app_backend/Untitled-2.mongodb

```diff
@@ -0,0 +1,65 @@
+// MongoDB Playground
```

Comment:
Either `.gitignore` this file or delete it. I don't see where you use it and think this was just for testing purposes?